packages/block-editor/src/layouts/flex.js

```diff
@@ -44,12 +44,15 @@ const alignItemsMap = {
 	left: 'flex-start',
 	right: 'flex-end',
 	center: 'center',
+	stretch: 'stretch',
```

Comment:
Do we also need to update the corresponding allowed values in `layout.php` around here: https://github.com/WordPress/gutenberg/blob/a41ea8177b7ac9c2b1861bbacf5768ec459195b0/lib/block-supports/layout.php#L190 ?

Reply:
I just pushed the update!